codegen/smithy-ruby-codegen-test-utils/src/main/resources/META-INF/services/software.amazon.smithy.ruby.codegen.RubyIntegration

```diff
@@ -1 +1,2 @@
+software.amazon.smithy.ruby.codegen.integrations.TestIntegration
```

Comment:
I feel like TestIntegration may no longer be the right name for this.  It only provides the "FakeProtocol" - maybe we should call it the FakeProtocolIntegration?

We may also want to rename FakeProtocol to something more descriptive like, NoOpProtocol or NullProtocol or something like that?

Reply:
Can do this in a different PR